### PR TITLE
Adds new step to help users setup keychain group

### PR DESCRIPTION
**Pull request by mendesbarreto**

### CODE REVIEW

**Comment by mendesbarreto** [comment]
Hello every one,

I was testing some features from MSAL and I had problem to make the request to acquire token

```swift
(NSOSStatusErrorDomain, -34018)
Error Domain=NSOSStatusErrorDomain Code=-34018 "(null)" UserInfo={MSALErrorDescriptionKey=Failed to set item into keychain.}
```
I suspected the SDK tries to save the auth information in the application using the keychain with some group default name. So I discovered you are using ```com.microsoft.adalcache``` and I put adicional step in the Readme file.

I think the step is valid because we can use "Application Single Sing On" when we have multiple application witch use the same login.

File: README.md [hunk 1]
```diff
@@ -183,6 +183,11 @@ If you find a security issue with our libraries or services please report it to
     </array>
 ```
 
+6. Add a new keychain group to project Capabilities `com.microsoft.adalcache`
```
Comment (oldalton) [inline README.md]: Thanks for adding this. I agree that it makes a lot of sense. I'd also prefer to link the detailed information here that explains why it is necessary. We have such information in our Wiki: [https://github.com/AzureAD/microsoft-authentication-library-for-objc/wiki/Keychain-on-iOS](https://github.com/AzureAD/microsoft-authentication-library-for-objc/wiki/Keychain-on-iOS).

Comment (mendesbarreto) [reply README.md]: Updated =)

File: README.md [hunk 2]
```diff
@@ -182,11 +182,15 @@ If you find a security issue with our libraries or services please report it to
         </dict>
     </array>
 ```
-
 See more info about configuring redirect uri for MSAL in our [Wiki](https://github.com/AzureAD/microsoft-authentication-library-for-objc/wiki/Redirect-uris-in-MSAL)
 
 Our library uses the ASWebAuthenticationSession for authentication on iOS 12 by default. See more information about default values, and support for other iOS versions [Wiki](https://github.com/AzureAD/microsoft-authentication-library-for-objc/wiki/MSAL-for-iOS-uses-web-browser)
 
+6. Add a new keychain group to your project Capabilities `com.microsoft.adalcache` . See more information about keychain groups for MSAL in our [Wiki](https://github.com/AzureAD/microsoft-authentication-library-for-objc/wiki/Keychain-on-iOS.)
```
Comment (oldalton) [inline README.md]: The "." in the end of the URL makes URL point to a non-existent page.
It should be (https://github.com/AzureAD/microsoft-authentication-library-for-objc/wiki/Keychain-on-iOS) without dot.

Comment (mendesbarreto) [reply README.md]: No problem ;)

**Comment by mendesbarreto** [comment]
I removed the unnecessary commits from this pull request. 

To clean the commits messages and make it easier to read.

**Pull request merged**